dokka-integration-tests/gradle/projects/README.md

```diff
@@ -0,0 +1,16 @@
+# Running integration tests projects locally
+
+Projects used to run integration tests could be opened in or build independently by following these steps:
+
+1. open `gradle.properties` file of a specific project and change commented variables:
+    1. `dokka_it_kotlin_version` to required version of Kotlin Gradle Plugin, f.e 2.0.0
+    2. `dokka_it_dokka_version` to required version to Dokka Gradle plugin, f.e 1.9.20
```

Comment:
nitpicking: `e.g.` is clearer than `f.e.`
formality: it seems the uncomment step is missed.